pipelines/windows-bios-installer/README.md

```diff
@@ -0,0 +1,127 @@
+2. Replace the default example autounattend.xml with your own in the definition of the `windows10-autounattend` ConfigMap in the pipeline YAML.
+   Different autounattend.xml can be also passed in a separate ConfigMap with the Pipeline parameter `autounattendConfigMapName` when creating a PipelineRun.
+
+## Pipeline Description
+
+```
+  create-vm-root-disk --- create-vm --- wait-for-vmi-status --- cleanup-vm
+```
+
+1. `create-vm-root-disk` task creates an empty DV.
+2. `create-vm` task creates a VM called `windows-bios-installer-*`
+   from the empty DV and with the `windows-bios-installer-cd-rom` DV attached as a CD-ROM.
+   A second DV with the virtio-win ISO will also be attached (Pipeline parameter `virtioContainerDiskName`).
+3. `wait-for-vmi-status` task waits until the VM shuts down.
+4. `cleanup-vm` deletes the installer VM and ISO DV (also in case of failure of the previous tasks).
+5. The output artifact will be the `win10` DV with the basic Windows installation.
+   It will boot into the Windows OOBE and needs to be setup further before it can be used.
+
+## How to run
+
+Pipeline runs with resolvers:
+```yaml
+apiVersion: tekton.dev/v1
+kind: PipelineRun
+metadata:
+    generateName: windows10-installer-run-
+spec:
+    params:
+    -   name: winImageDownloadURL
+        value: INSERT_WINDOWS_ISO_URL
+    pipelineRef:
+        params:
+        -   name: catalog
+            value: kubevirt-tekton-pipelines
+        -   name: type
+            value: artifact
+        -   name: kind
+            value: task
+        -   name: name
+            value: windows-bios-installer
+        -   name: version
+            value: v0.18.0
+        resolver: hub
+    serviceAccountName: pipeline
+---
+```
+
+### Usage in multiple namespaces
+
+When user defines a different namespace in e.g. `baseDvNamespace` parameter than the pipelinerun is running, the serviceAccount under which pipeline is running, will require additional permissions. Required permissions to run task in different namespace can be found in README of each task.
+
+## Possible Optimizations
+
+### Windows Source ISO Caching
+
+Windows source ISO can be downloaded and cached/stored in a DV.
+So the subsequent PipelineRuns don't have to download the same ISO multiple times.
+
+You can for example create the following DV first and remove create-source-dv step from the windows10-installer pipeline:
+
+```yaml
+ apiVersion: cdi.kubevirt.io/v1beta1
+kind: DataVolume
+metadata:
+  name: windows10-source
+spec:
+  pvc:
+    accessModes:
+      - ReadWriteOnce
+    resources:
+      requests:
+        storage: 7Gi
+    volumeMode: Filesystem
+  source:
+    http:
+      url: WIN_IMAGE_DOWNLOAD_URL
+```
+## Cancelling/Deleteting pipelineRuns
+
+When running the example pipelines, they create temporary objects (DataVolumes, VMs, templates, etc.). Each pipeline has its own clean up system which 
```

Comment:
Stick to one way of spelling acronyms or names: DataVolumes, VirtualMachines, Templates?